### ISSUE TITLE
Team import doesn't set the organisation

### ISSUE THREAD

**Issue by lyz-code** [Lerentis/terraform-provider-gitea#35]
**Describe the bug**
When importing a team the required `organisation` field is not set. So when you try to do a `plan` the resource gets replaced as it's trying to change from `""` to the value you want to set.

**To Reproduce**
1. Import a team
2. Do a `terraform plan`

**Expected behavior**
When importing an organisation team, the `organisation` field should be set in the state.

**Log Output**
If applicable, add logs to help explain your problem.

**Additional Data**
Important for reproducability.

- Terraform Version: v0.13.1

- Operating System: Linux

- Provider Version: v0.12.3

**Comment by lyz-code** [comment]
It seems that the team members are not imported either, this is not that bad because in the first apply they are overwritten

**Comment by Lerentis** [comment]
Hi @lyz-code ,

i can confirm your observation and in theory the fix for it should be easy, but there seems to be a bug in the gitea api where a `GET /teams/{id}` returns `null` for the Organisation field... 

```json
{
  "id": 13,
  "name": "test_team",
  "description": "test team of test org",
  "organization": null,
  "includes_all_repositories": true,
  "permission": "admin",
  "units": [
    "repo.releases",
    "repo.projects",
    "repo.packages",
    "repo.pulls",
    "repo.wiki",
    "repo.ext_wiki",
    "repo.code",
    "repo.issues",
    "repo.ext_issues"
  ],
  "units_map": {
    "repo.code": "admin",
    "repo.ext_issues": "admin",
    "repo.ext_wiki": "admin",
    "repo.issues": "admin",
    "repo.packages": "admin",
    "repo.projects": "admin",
    "repo.pulls": "admin",
    "repo.releases": "admin",
    "repo.wiki": "admin"
  },
  "can_create_org_repo": true
}
```

this needs to be fixed upstream i fear

**Comment by Lerentis** [comment]
Reported upstream here: https://github.com/go-gitea/gitea/issues/24734

if you would like to add anything to my description feel free to add your comment there 😃

**Comment by lyz-code** [comment]
Thank you for taking care of it, and damn they are fast fixing bugs!

**Comment by Lerentis** [comment]
indeed 😃 now just need to wait until this is released and i need to check the version where the fix is present in order to not break the functionality for older gitea releases. will update this issue when i release the fix.
thanks again for the report

**Comment by Lerentis** [comment]
missed the gitea release but it should now be fixed and released as [v0.14.1](https://github.com/Lerentis/terraform-provider-gitea/releases/tag/v0.14.1)
